lib/experimental/Widgets/Content/CopyLabel/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { Icon } from "@/components/Utilities/Icon"
+import { Check, LayersFront } from "@/icons"
+import { cn } from "@/lib/utils"
+import { forwardRef, useEffect, useState } from "react"
+interface CopyLabelType {
+  text: string
+}
+
+export const CopyLabel = forwardRef<HTMLDivElement, CopyLabelType>(
+  ({ text }, ref) => {
+    const [copied, setCopied] = useState(false)
+
+    useEffect(() => {
+      if (copied) {
+        const timer = setTimeout(() => setCopied(false), 1500)
```

Comment:
I'd extracted the delay to a constant, like `const COPIED_SHOWN_MS = 1500`

Have you tried 1s or even 0.75s? 1.5s looks a bit long for my taste. If user missed the notification, one more click is cheap to make

Reply:
A matter of taste I guess, I tried 1000 first, but I liked it more at 1500. I'll extract the delay.